server/services/approverCaseloadService.test.ts

```diff
@@ -116,7 +116,7 @@ describe('Approval Caseload Service', () => {
     const expectedResult = {
       licenceId: 1,
       approvedBy: 'Jim Smith',
-      approvedOn: '25 April 2014',
+      approvedOn: '25 Apr 2014',
```

Comment:
after talking to UCD I made this consistent so both types of date on the approver case list table use the same format